gulpfile.js

```diff
@@ -192,33 +218,26 @@ gulp.task('styles-grid', function() {
     .pipe($.size({title: 'styles-grid'}));
 });
 
+// Build with Google's Closure Compiler.
+// To run this step, run "bower install" within MDL to fetch the compiler jar,
+// note that Java 1.7+ is required. This is an optional step.
+gulp.task('closure', function() {
+  return gulp.src(SOURCES)
+    .pipe(closureCompiler({
+      compilerPath: 'bower_components/closure-compiler/compiler.jar',
```

Comment:
I thought Closure could only be downloaded with Bower, but turns out there's a `google-closure-compiler` NPM module. I've depended on that instead.

Note that the `closure-compiler` module isn't encouraged by us - we encourage `google-closure-compiler` (linked to from [google/closure-compiler](https://github.com/google/closure-compiler)).